### PR TITLE
Added config settings and config form

### PR DESCRIPTION
**Pull request by cam8001**

### CODE REVIEW

File: config/install/akamai.settings.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+akamai_disabled: 1
+akamai_restapi_endpoint: https://api.ccu.akamai.com/ccu/v2/queues/default
+akamai_basepath: null
+akamai_timeout: 5
+akamai_username: null
+akamai_password: null
+akamai_domain:
+  - staging
```
Comment (nerdstein) [inline config/install/akamai.settings.yml]: I don't think "Action" or "Domain" is a sequence, I think its a string selection of one of these. We can set a sane default upon install at your discretion

File: config/install/akamai.settings.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  - production
+akamai_action:
+  - remove
```
Comment (nerdstein) [inline config/install/akamai.settings.yml]: See comment above

File: src/Form/AkamaiConfigForm.php [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+   */
+  public function submitform(array &$form, FormStateInterface $form_state) {
+    $this->config('akamai.settings')
+      ->set('akamai_disabled', $form_state->getValue('akamai_disabled'))
+      ->set('akamai_restapi_endpoint', $form_state->getValue('akamai_restapi_endpoint'))
+      ->set('akamai_basepath', $form_state->getValue('akamai_basepath'))
+      ->set('akamai_timeout', $form_state->getValue('akamai_timeout'))
+      ->set('akamai_username', $form_state->getValue('akamai_username'))
+      ->set('akamai_password', $form_state->getValue('akamai_password'))
+      ->set('akamai_domain', $form_state->getValue('akamai_password'))
```
Comment (nerdstein) [inline src/Form/AkamaiConfigForm.php]: Copy pasta error - "akamai_password"? see domain and action settings

**Comment by cam8001** [comment]
Should be good now - let me know :)

**Comment by nerdstein** [comment]
Awesome

**Pull request merged**